feature_config.json

```diff
@@ -232,19 +232,19 @@
       ]
     },
     {
-      "name": "Is Pokemon id Features",
-      "path": "./feature_sources/isPokemonId_features.js",
+      "name": "Terrain Type Feature",
+      "path": "./feature_sources/terrain_features.js",
       "enabled": true,
       "features": [
         {
           "key": "terrainType",
           "type": "numerical",
```

Comment:
Thanks for the updates! :) Numerical though, should be still replaced by numeric ;)